test/90_1_prm/test_create_prototype_baseline_building.rb

```diff
@@ -37,21 +46,343 @@ def self.generate_prototype_model_and_baseline(building_type, template, climate_
+        'SecondarySchoolCorridor' => 'corridor - all other',
+        'SecondarySchoolGym' => 'gymnasium playing area',
+        'SecondarySchoolLibrary' => 'library - whole building',
+        'SecondarySchoolLobby' => 'lobby - all other',
+        'SecondarySchoolOffice' => 'office - enclosed <= 250 sf',
+        'SecondarySchoolRestroom' => 'restroom - all other',
+        'SecondarySchoolKitchen' => 'kitchen',
+        'SecondarySchoolMechanical' => 'electrical/mechanical',
+        'SmallDataCenterHighITEComputerRoom' => 'office - open',
+        'SmallDataCenterLowITEComputerRoom' => 'office - open',
+        'SmallHotelCorridor' => 'corridor - all other',
+        'SmallHotelCorridor4' => 'corridor - all other',
+        'SmallHotelElec/MechRoom' => 'electrical/mechanical',
+        'SmallHotelElevatorCore' => 'elevator core',
+        'SmallHotelElevatorCore4' => 'elevator core',
+        'SmallHotelExercise' => 'gymnsasium exercise area',
+        'SmallHotelGuestRoom123Occ' => 'guest room',
+        'SmallHotelGuestRoom123Vac' => 'guest room',
+        'SmallHotelGuestRoom4Occ' => 'guest room',
+        'SmallHotelGuestRoom4Vac' => 'guest room',
+        'SmallHotelLaundry' => 'laundry/washing',
+        'SmallHotelMechanical' => 'electrical/mechanical',
+        'SmallHotelMeeting' => 'conference/meeting/multipurpose',
+        'SmallHotelOffice' => 'office - enclosed <= 250 sf',
+        'SmallHotelPublicRestroom' => 'restroom - all other',
+        'SmallHotelStaffLounge' => 'lounge/breakroom - all other',
+        'SmallHotelStair' => 'stairwell',
+        'SmallHotelStair4' => 'stairwell',
+        'SmallHotelStorage' => 'storage 50 to 1000 sf - all other',
+        'SmallHotelStorage4Front' => 'storage 50 to 1000 sf - all other',
+        'SmallHotelStorage4Rear' => 'storage 50 to 1000 sf - all other',
+        'SmallHotelGuestLounge' => 'lobby - hotel',
+        'StripMallStrip mall - type 1' => 'retail - whole building',
+        'StripMallStrip mall - type 2' => 'retail - whole building',
+        'StripMallStrip mall - type 3' => 'retail - whole building',
+        'SuperMarketMeeting' => 'conference/meeting/multipurpose',
+        'SuperMarketDining' => 'dining - cafeteria/fast food',
+        'SuperMarketRestroom' => 'restroom - all other',
+        'SuperMarketElec/MechRoom' => 'electrical/mechanical',
+        'SuperMarketCorridor' => 'corridor - all other',
+        'SuperMarketDeli' => 'kitchen',
+        'SuperMarketBakery' => 'kitchen',
+        'SuperMarketVestibule' => 'lobby - all other',
+        'SuperMarketSales' => 'sales',
+        'SuperMarketProduce' => 'sales',
+        'SuperMarketDryStorage' => 'storage 50 to 1000 sf - all other',
+        'SuperMarketOffice' => 'office - enclosed <= 250 sf',
+        'WarehouseOffice' => 'office - enclosed <= 250 sf',
+        'WarehouseFine' => 'warehouse - fine storage',
+        'WarehouseBulk' => 'warehouse - bulk storage'
+      }
+
```

Comment:
What do you think about putting all of these information in a JSON file to make the script smaller?